CONTRIBUTING.txt

```diff
@@ -317,7 +317,19 @@ Building docs
 -------------
 
 To build docs, run ``make`` from the ``doc`` directory. ``make help`` lists
-all targets.
+all targets. For example, to build the HTML documentation, you can run:
+
+.. code:: sh
+
+    make SPHINXOPTS= html
```

Comment:
Not just `make html` ?

Reply:
If I just run `make html`, I got the following error on my mac:
```
python tools/build_modref_templates.py
/Users/thein/hacktoberfest/scikit-image/skimage/novice/__init__.py:103: UserWarning: The `skimage.novice` module was deprecated in version 0.14. It will be removed in 0.16.
  warnings.warn("The `skimage.novice` module was deprecated in version 0.14. "
WARNING: Empty - skimage.scripts
outdir:
27 files written
Build API docs...done.
Copying release notes
make: nproc: Command not found
python -m sphinx -b html -d build/doctrees  -j  source build/html
usage: __main__.py [OPTIONS] SOURCEDIR OUTPUTDIR [FILENAMES...]
__main__.py: error: argument -j: invalid jobs_argument value: 'source'
make: *** [html] Error 2
```